deploy/ray-operator/utils/util.go

```diff
@@ -0,0 +1,28 @@
+package utils
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"strconv"
+	"strings"
+)
+
+// The function isCreated returns true if pod has been created and is maintained by the API server
+func IsCreated(pod *corev1.Pod) bool {
+	return pod.Status.Phase != ""
+}
+
+func Before(value string, a string) string {
```

Comment:
```suggestion
func substringBefore(value string, a string) string {
```

Reply:
In Go, capital letter means the function can be accessed by other package, lower-case means private can only be accessed in the same package. I will update the function name

Reply:
Sure. Please pay attention that not every method begins with a capital letter has been suggested to change to the lowercase letter. So, in this way, the suggestion focused actually on the design idea about the visibility of helper methods:
https://stackoverflow.com/questions/38616687/which-way-to-name-a-function-in-go-camelcase-or-semi-camelcase